packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -254,21 +274,28 @@ export class DebugConfigurationManager {
 
     protected async doOpen(model: DebugConfigurationModel): Promise<EditorWidget> {
         let uri = model.uri;
-        if (!uri) {
+        const configFileUri = this.getFolderConfigurationUri(model);
```

Comment:
I think this sequence is ignoring the case where there _is_ a launch.json but it's in the .vscode folder instead of .theia